src/assets/simp-environment/environments/simp/hieradata/scenarios/poss.yaml

```diff
@@ -14,4 +14,4 @@
 # NOTE: Details for the 'poss' scenario can be found in the
 # 'simp::scenario::poss' class
 
-classes : []
+simp::scenario: 'poss'
```

Comment:
@lnemsick-simp and @jeefberkey This *is* currently needed due to moving the simp include back to an `include` statement. The old way made it so that it simply could not be set via Hiera for different nodes and that would not work.